src/js/Keyboard.js

```diff
@@ -29,12 +29,17 @@ class Keyboard extends Component {
   }
 
   handleAutoComplete = append => {
```

Comment:
```suggestion
  handleAutoComplete = newInput => {
```
Should rename the variable appropriately of course